static/src/inline-svgs/logo/the-guardian-logo.svg

```diff
@@ -1,3 +1 @@
s="http://www.w3.org/2000/svg" viewBox="0 0 297 95"><path fill="#121212" d="M66.8 50.69l5.04-2.63V8.43h-3.81L58.7 20.79h-1.05l.58-13.78h40.46l.59 13.78h-1.12L89.03 8.43h-3.92v39.55l5.08 2.68V52H66.8v-1.31zm37.09-1.76V5L100 3.45v-.82L114.16.08h1.5v20.79l.39-.33A19 19 0 0 1 128.2 16c6.21 0 9 3.5 9 10v22.93l3.32 1.8V52h-18.59v-1.25l3.33-1.81V26c0-3.59-1.57-5-4.51-5a7.76 7.76 0 0 0-4.9 1.62V49l3.27 1.8V52h-18.56v-1.19zm48.31-13.48c.39 7.26 3.64 12.88 11.36 12.88 3.73 0 6.38-1.73 8.88-3v1.43A17.36 17.36 0 0 1 158.8 53c-12 0-18.08-6.64-18.08-18.14 0-11.24 6.69-18.24 17.5-18.24 10.17 0 15.44 5.07 15.44 18.43v.35zm-.2-1.67l10.53-.64c0-9-1.54-15-4.63-15-3.28 0-5.9 6.93-5.9 15.6M0 69.58c0-19.11 12.67-25.92 26.8-25.92 6 0 11.64 1 14.81 2.27l.28 13.34h-1.36l-8.29-12.91a12.18 12.18 0 0 0-5.24-.84c-7.51 0-11.34 8.66-11.22 22.85.14 17 3.1 24.69 10 24.69a10.71 10.71 0 0 0 4.13-.69V74.19l-4.55-2.6V70h22v1.69l-4.48 2.5v17.95A49.21 49.21 0 0 1 26.24 95C10.19 95 0 87.53 0 69.58m47.11-8.92v-1.1L61.89 57l1.62.13v29.03c0 3.5 1.69 4.58 4.52 4.58a6 6 0 0 0 4.79-2.26V62.42l-4.05-1.75v-1.15l14.78-2.6 1.49.14v33.31L89 92v1.1l-14.56 1.79-1.44-.13v-4.38h-.41A16.39 16.39 0 0 1 61.48 95c-7.09 0-10.33-4.18-10.33-10.51V62.42zm94.51-3.76l1.21.14v10.75h.34c1.58-7.89 5.06-10.79 9.31-10.79a4.7 4.7 0 0 1 1.82.27v11a12.62 12.62 0 0 0-3-.27 18.58 18.58 0 0 0-8 1.6v21.24l3.36 1.86V94h-19.22v-1.29l3.45-1.87v-29l-4-1.21v-1z"/><path d="M178.72 57.8V46.45l-4-1.42v-.91l14.92-2.74 1.41.2v48.78l4.13 1.49v1.25L180.42 95l-1.15-.13v-4h-.34a13.14 13.14 0 0 1-9.79 4.13c-8 0-13.91-6.13-13.91-18.66 0-13.21 6.82-19.7 17.14-19.7a14.65 14.65 0 0 1 6.34 1.15m0 31.22V59.88a5.45 5.45 0 0 0-4.08-1.27c-4 .14-6.45 6.15-6.45 16.87 0 9.63 1.77 15 7.09 14.84A5.23 5.23 0 0 0 178.7 89m32.82-32.14l1.28.13v33.85l3.37 1.86V94H197v-1.29l3.44-1.87V62.35l-4.12-1.62V59.6zm1.35-9.14a6.41 6.41 0 0 1-6.55 6.28 6.3 6.3 0 1 1 0-12.6 6.46 6.46 0 0 1 6.55 6.33m46.34 43.11V62l-4.05-1.42v-1.4l14.71-2.74 1.49.13v4.31h.4a19.44 19.44 0 0 1 12.56-4.65c6.41 0 9.25 3 9.25 9.77v24.79L297 92.7V94h-19.16v-1.29l3.44-1.87V66.73c0-3.7-1.62-5.19-4.66-5.19a8.06 8.06 0 0 0-5.06 1.61v27.68l3.38 1.87V94h-19.18v-1.29zm-21.26-18.1v-4.83c0-7.28-1.59-9.66-6.1-9.66a11.82 11.82 0 0 0-1.53.13l-8 10.85h-1.13v-10a43.28 43.28 0 0 1 13.46-2.32c9.81 0 15.52 2.71 15.52 10.92V91.4l3.51.93v.93a15.08 15.08 0 0 1-7.23 1.65c-4.84 0-7.16-1.59-8.22-4.24h-.33c-2.06 2.78-5 4.37-9.55 4.37-5.84 0-9.81-3.64-9.81-9.93 0-6.09 3.78-9.4 11.47-10.85zm0 16.28V74.46l-2.45.2c-3.85.33-5.24 2.78-5.24 8.21 0 5.89 1.92 7.41 4.64 7.41a3.57 3.57 0 0 0 3-1.26M109.69 72.74v-4.83c0-7.28-1.59-9.66-6.1-9.66a11.79 11.79 0 0 0-1.53.13L94 69.23h-1.09v-10a43.29 43.29 0 0 1 13.46-2.32c9.81 0 15.52 2.71 15.52 10.92V91.4l3.51.93v.93a15.08 15.08 0 0 1-7.23 1.65c-4.84 0-7.16-1.59-8.22-4.24h-.33c-2.06 2.78-5 4.37-9.55 4.37-5.83 0-9.81-3.64-9.81-9.93 0-6.09 3.78-9.4 11.47-10.85zm0 16.28V74.46l-2.45.2c-3.85.33-5.24 2.78-5.24 8.21 0 5.89 1.92 7.41 4.64 7.41a3.57 3.57 0 0 0 3-1.26" fill="#121212"/></svg>
```

Comment:
want to add teh `xmlns="http://www.w3.org/2000/svg"`?

Reply:
ugh, ignore me! sorry @zeftilldeath!

Reply:
can make a slight saving i think? see what you reckon:

```xml
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 297 95">
  <path fill="#121212" d="M66.8 50.7l5-2.6V8.4H68l-9.3 12.4h-1L58.2 7h40.5l.6 13.8h-1.1L89 8.4h-3.9V48l5 2.7V52H66.9v-1.3zm37-1.8V5L100 3.5v-.9L114.2.1h1.5v20.8l.3-.4a19 19 0 0 1 12.2-4.5c6.2 0 9 3.5 9 10v23l3.3 1.7V52H122v-1.3l3.4-1.8V26c0-3.6-1.6-5-4.6-5a7.8 7.8 0 0 0-4.9 1.6V49l3.3 1.8V52h-18.5v-1.2zm48.4-13.4c.4 7.2 3.6 12.8 11.4 12.8 3.7 0 6.3-1.7 8.8-3v1.5a17.4 17.4 0 0 1-13.6 6.2c-12 0-18-6.6-18-18.1 0-11.3 6.6-18.3 17.4-18.3 10.2 0 15.5 5 15.5 18.4v.4zm-.2-1.7l10.5-.7c0-9-1.5-15-4.6-15-3.3 0-5.9 7-5.9 15.6M0 69.6c0-19.1 12.7-26 26.8-26 6 0 11.6 1 14.8 2.3l.3 13.4h-1.4l-8.3-13a12.2 12.2 0 0 0-5.2-.8c-7.5 0-11.3 8.7-11.2 22.9.1 17 3 24.7 10 24.7a10.7 10.7 0 0 0 4.1-.7V74.2l-4.5-2.6V70h22v1.7l-4.5 2.5v18A49.2 49.2 0 0 1 26.2 95C10.2 95 0 87.5 0 69.6m47.1-9v-1L62 57l1.6.1v29c0 3.6 1.7 4.6 4.5 4.6a6 6 0 0 0 4.8-2.2v-26l-4-1.8v-1.2L83.6 57l1.4.2v33.3l4 1.6v1.1L74.4 95l-1.4-.1v-4.4h-.4A16.4 16.4 0 0 1 61.5 95C54.4 95 51 90.8 51 84.5v-22zm94.5-3.7l1.2.1v10.8h.4c1.6-7.9 5-10.8 9.3-10.8a4.7 4.7 0 0 1 1.8.3v11a12.6 12.6 0 0 0-3-.3 18.6 18.6 0 0 0-8 1.6v21.2l3.4 1.9V94h-19.3v-1.3l3.5-1.9v-29l-4-1.2v-1z M178.7 57.8V46.5l-4-1.5v-.9l15-2.7 1.3.2v48.8l4.2 1.5V93l-14.8 2-1.1-.1v-4h-.4a13.1 13.1 0 0 1-9.8 4.1c-8 0-13.9-6.1-13.9-18.7 0-13.2 6.9-19.7 17.2-19.7a14.7 14.7 0 0 1 6.3 1.2m0 31.2V60a5.5 5.5 0 0 0-4-1.3c-4 .1-6.5 6.2-6.5 16.9 0 9.6 1.8 15 7 14.8a5.2 5.2 0 0 0 3.5-1.3M211.5 57h1.3v34l3.4 1.8V94H197v-1.3l3.4-1.9V62.4l-4-1.7v-1.1zm1.4-9.2a6.4 6.4 0 0 1-6.6 6.3 6.3 6.3 0 1 1 0-12.6 6.5 6.5 0 0 1 6.6 6.3m46.3 43.1V62l-4-1.4v-1.4l14.7-2.8 1.5.2v4.3h.4a19.4 19.4 0 0 1 12.5-4.7c6.4 0 9.3 3 9.3 9.8v24.8l3.4 1.9V94h-19.2v-1.3l3.5-1.9v-24c0-3.8-1.6-5.3-4.7-5.3a8 8 0 0 0-5 1.7v27.6l3.3 1.9V94h-19.1v-1.3zm-21.3-18V68c0-7.3-1.5-9.6-6-9.6a11.8 11.8 0 0 0-1.6 0l-8 11h-1.1v-10a43.3 43.3 0 0 1 13.5-2.4c9.8 0 15.5 2.7 15.5 11v23.5l3.5 1v.9a15 15 0 0 1-7.2 1.6c-4.9 0-7.2-1.6-8.3-4.2h-.3c-2 2.8-5 4.3-9.6 4.3-5.8 0-9.8-3.6-9.8-9.9 0-6 3.8-9.4 11.5-10.8zm0 16.2V74.5l-2.4.2c-3.9.3-5.2 2.7-5.2 8.2 0 5.9 1.9 7.4 4.6 7.4a3.6 3.6 0 0 0 3-1.3M109.7 72.7V68c0-7.3-1.6-9.7-6.1-9.7a11.8 11.8 0 0 0-1.5.2L94 69.2h-1v-10a43.3 43.3 0 0 1 13.4-2.3c9.8 0 15.5 2.7 15.5 11v23.5l3.5 1v.9a15 15 0 0 1-7.2 1.6c-4.9 0-7.2-1.6-8.3-4.2h-.3c-2 2.8-5 4.3-9.5 4.3-5.9 0-9.8-3.6-9.8-9.9 0-6 3.7-9.4 11.4-10.8zm0 16.3V74.5l-2.5.2c-3.8.3-5.2 2.7-5.2 8.2 0 5.9 2 7.4 4.6 7.4a3.6 3.6 0 0 0 3-1.3" fill="#121212" />
</svg>
```

Reply:
Thanks @sndrs how are you doing these?

Reply:
using https://jakearchibald.github.io/svgomg and then just merging the paths by hand

Reply:
@sndrs @zeftilldeath This last one does not display for me in FF and Chrome:
`error on line 2 at column 2418: Attribute fill redefined`

Removing a duplicate `fill`, because paths were merged, works:
```
<svg data-name="The Guardian" viewBox="0 0 295 97" xmlns="http://www.w3.org/2000/svg"> width="295" height="97"><path fill="#121212" d="M66.8 50.7l5-2.6V8.4H68l-9.3 12.4h-1L58.2 7h40.5l.6 13.8h-1.1L89 8.4h-3.9V48l5 2.7V52H66.9v-1.3zm37-1.8V5L100 3.5v-.9L114.2.1h1.5v20.8l.3-.4a19 19 0 0 1 12.2-4.5c6.2 0 9 3.5 9 10v23l3.3 1.7V52H122v-1.3l3.4-1.8V26c0-3.6-1.6-5-4.6-5a7.8 7.8 0 0 0-4.9 1.6V49l3.3 1.8V52h-18.5v-1.2zm48.4-13.4c.4 7.2 3.6 12.8 11.4 12.8 3.7 0 6.3-1.7 8.8-3v1.5a17.4 17.4 0 0 1-13.6 6.2c-12 0-18-6.6-18-18.1 0-11.3 6.6-18.3 17.4-18.3 10.2 0 15.5 5 15.5 18.4v.4zm-.2-1.7l10.5-.7c0-9-1.5-15-4.6-15-3.3 0-5.9 7-5.9 15.6M0 69.6c0-19.1 12.7-26 26.8-26 6 0 11.6 1 14.8 2.3l.3 13.4h-1.4l-8.3-13a12.2 12.2 0 0 0-5.2-.8c-7.5 0-11.3 8.7-11.2 22.9.1 17 3 24.7 10 24.7a10.7 10.7 0 0 0 4.1-.7V74.2l-4.5-2.6V70h22v1.7l-4.5 2.5v18A49.2 49.2 0 0 1 26.2 95C10.2 95 0 87.5 0 69.6m47.1-9v-1L62 57l1.6.1v29c0 3.6 1.7 4.6 4.5 4.6a6 6 0 0 0 4.8-2.2v-26l-4-1.8v-1.2L83.6 57l1.4.2v33.3l4 1.6v1.1L74.4 95l-1.4-.1v-4.4h-.4A16.4 16.4 0 0 1 61.5 95C54.4 95 51 90.8 51 84.5v-22zm94.5-3.7l1.2.1v10.8h.4c1.6-7.9 5-10.8 9.3-10.8a4.7 4.7 0 0 1 1.8.3v11a12.6 12.6 0 0 0-3-.3 18.6 18.6 0 0 0-8 1.6v21.2l3.4 1.9V94h-19.3v-1.3l3.5-1.9v-29l-4-1.2v-1z M178.7 57.8V46.5l-4-1.5v-.9l15-2.7 1.3.2v48.8l4.2 1.5V93l-14.8 2-1.1-.1v-4h-.4a13.1 13.1 0 0 1-9.8 4.1c-8 0-13.9-6.1-13.9-18.7 0-13.2 6.9-19.7 17.2-19.7a14.7 14.7 0 0 1 6.3 1.2m0 31.2V60a5.5 5.5 0 0 0-4-1.3c-4 .1-6.5 6.2-6.5 16.9 0 9.6 1.8 15 7 14.8a5.2 5.2 0 0 0 3.5-1.3M211.5 57h1.3v34l3.4 1.8V94H197v-1.3l3.4-1.9V62.4l-4-1.7v-1.1zm1.4-9.2a6.4 6.4 0 0 1-6.6 6.3 6.3 6.3 0 1 1 0-12.6 6.5 6.5 0 0 1 6.6 6.3m46.3 43.1V62l-4-1.4v-1.4l14.7-2.8 1.5.2v4.3h.4a19.4 19.4 0 0 1 12.5-4.7c6.4 0 9.3 3 9.3 9.8v24.8l3.4 1.9V94h-19.2v-1.3l3.5-1.9v-24c0-3.8-1.6-5.3-4.7-5.3a8 8 0 0 0-5 1.7v27.6l3.3 1.9V94h-19.1v-1.3zm-21.3-18V68c0-7.3-1.5-9.6-6-9.6a11.8 11.8 0 0 0-1.6 0l-8 11h-1.1v-10a43.3 43.3 0 0 1 13.5-2.4c9.8 0 15.5 2.7 15.5 11v23.5l3.5 1v.9a15 15 0 0 1-7.2 1.6c-4.9 0-7.2-1.6-8.3-4.2h-.3c-2 2.8-5 4.3-9.6 4.3-5.8 0-9.8-3.6-9.8-9.9 0-6 3.8-9.4 11.5-10.8zm0 16.2V74.5l-2.4.2c-3.9.3-5.2 2.7-5.2 8.2 0 5.9 1.9 7.4 4.6 7.4a3.6 3.6 0 0 0 3-1.3M109.7 72.7V68c0-7.3-1.6-9.7-6.1-9.7a11.8 11.8 0 0 0-1.5.2L94 69.2h-1v-10a43.3 43.3 0 0 1 13.4-2.3c9.8 0 15.5 2.7 15.5 11v23.5l3.5 1v.9a15 15 0 0 1-7.2 1.6c-4.9 0-7.2-1.6-8.3-4.2h-.3c-2 2.8-5 4.3-9.5 4.3-5.9 0-9.8-3.6-9.8-9.9 0-6 3.7-9.4 11.4-10.8zm0 16.3V74.5l-2.5.2c-3.8.3-5.2 2.7-5.2 8.2 0 5.9 2 7.4 4.6 7.4a3.6 3.6 0 0 0 3-1.3"/></svg>
```

Much better kerning BTW 👍